recipe/meta.yaml

```diff
@@ -37,28 +37,52 @@ requirements:
     - libprotobuf {{ libprotobuf }}
     - libabseil {{ libabseil }}
     - pytest-runner 6.0.0
-    - numpy   1.21  # [py<310]
-    - numpy   1.23  # [py==310 or py==311]
-    - numpy   1.26  # [py>=312]
+    - numpy 2.0 # [py<313]
+    - numpy 2.1 # [py==313]
   run:
     - python
     - protobuf >=3.20.2
-    - {{ pin_compatible('numpy') }}
+    - numpy >=1.20,<2.3
```

Comment:
Where is `<2.3` coming from?

Reply:
I updated it as `numpy >=1.20`, https://github.com/onnx/onnx/blob/v1.17.0/requirements.txt#L1